src/main/java/redis/clients/jedis/util/SafeEncoder.java

```diff
@@ -67,4 +69,12 @@ public static Object encodeObject(Object dataToEncode) {
 
     return dataToEncode;
   }
+
+  public static Map<byte[], byte[]> encodeStringMap(Map<String, String> map) {
+    final Map<byte[], byte[]> bhash = new HashMap<>(map.size());
+    for (final Map.Entry<String, String> entry : map.entrySet()) {
+      bhash.put(SafeEncoder.encode(entry.getKey()), SafeEncoder.encode(entry.getValue()));
+    }
+    return bhash;
+  }
```

Comment:
Move this in Client class as private static method. We'll add this here when we more usages.